package.json

```diff
@@ -17,22 +17,23 @@
     "@commitlint/cli": "^16.0.2",
     "@commitlint/config-conventional": "^16.0.0",
     "@commitlint/config-lerna-scopes": "^16.0.0",
-    "@typescript-eslint/eslint-plugin": "^5.15.0",
-    "@typescript-eslint/parser": "^5.15.0",
     "conventional-changelog-conventionalcommits": "^4.6.3",
-    "eslint": "^8.11.0",
-    "eslint-config-prettier": "^8.5.0",
-    "eslint-plugin-simple-import-sort": "^7.0.0",
-    "husky": "^7.0.4",
+    "eslint": "^8.24.0",
+    "eslint-config-custom": "*",
+    "husky": "^8.0.1",
     "lerna": "^4.0.0",
     "prettier": "^2.6.0",
     "pretty-quick": "^3.1.3",
     "rimraf": "^3.0.2",
-    "turbo": "^1.1.5",
+    "turbo": "^1.5.5",
     "typescript": "^4.6.4"
   },
   "scripts": {
-    "dev": "lerna run dev --parallel --ignore @clerk/{expo}",
```

Comment:
❓ I guess Lerna wasn't required at this point, right?

Reply:
Yup, right now lerna is only needed for the release management at the moment.